go/vt/vttablet/tabletserver/state_manager.go

```diff
@@ -666,30 +666,32 @@ func (sm *stateManager) Broadcast() {
 }
 
 func (sm *stateManager) refreshReplHealthLocked() (time.Duration, error) {
-	if sm.target.TabletType == topodatapb.TabletType_PRIMARY {
-		sm.replHealthy = true
-		return 0, nil
-	}
-	lag, err := sm.rt.Status()
-	if err != nil {
-		if sm.replHealthy {
-			log.Infof("Going unhealthy due to replication error: %v", err)
-		}
-		sm.replHealthy = false
-	} else {
-		if lag > sm.unhealthyThreshold.Get() {
-			if sm.replHealthy {
-				log.Infof("Going unhealthy due to high replication lag: %v", lag)
-			}
-			sm.replHealthy = false
-		} else {
-			if !sm.replHealthy {
-				log.Infof("Replication is healthy")
-			}
-			sm.replHealthy = true
-		}
-	}
-	return lag, err
+	sm.replHealthy = true
+	return 0, nil
+	//if sm.target.TabletType == topodatapb.TabletType_PRIMARY {
```

Comment:
it's not good to comment out the code, either add comment the explain the reason why disable these lines temporarily or delete it completely if the code won't be used any longer.

Reply:
add comment